apis/secretsmanager/v1beta1/custom_types.go

```diff
@@ -88,4 +90,8 @@ type SecretReference struct {
 	// Key whose value will be used. If not given, the whole map in the Secret
 	// data will be used.
 	Key *string `json:"key,omitempty"`
+
+	// Type of the secret. Used to (re)create k8s secret in case of loss.
+	// If not given, the controller will try to fetch the type from the referenced secret.
+	Type *string `json:"type,omitempty"`
```

Comment:
The description for the newly added field was rewritten to reflect the controller lateInitialize behaviour for it.